sites/fast-website/src/app/components/content-placement-container/content-placement-container.styles.ts

```diff
@@ -48,20 +56,27 @@ export const ContentPlacementContainerStyles = css`
 
     /* This creates the color, background, and elevation changes on hover */
 
-    :host([section="feature"]:hover) site-card-section,
+    :host([section="feature"]:hover),
     :host([section="community"]:hover) site-content-placement,
     :host([section="community"]:hover) site-content-placement ::part(content) {
         color: var(--neutral-foreground-hint);
     }
 
+    :host([section="feature"]:hover) site-feature-card {
+        filter: saturate(0);
+    }
+
     :host([section="feature"]) site-feature-card:hover {
         color: var(--neutral-foreground-rest);
+        background: var(--neutral-fill-focus);
+        cursor: pointer;
+        filter: saturate(1);
     }
 
     :host([section="community"]) site-content-placement:hover {
         --elevation: 4;
         cursor: pointer;
-        background: var(--neutral-fill-active);
+        background: var(--neutral-fill-focus);
```

Comment:
With this change should the behaviors/imports also be updated with removing active and adding focus?

Reply:
I think that PR #3256 will be changing the way that we use the recipes so this might be something that we address later.

Reply:
As part of this PR, I had to reimplement the hover for the feature card background.  When I talked with Jesse the other day he indicated he had a darker black in mind for the background, and while we were talking we found that neutral-fill-focus is close to what he had in the design.  Also, after checking with some accessibility tools it became apparent that the lighter background of neutral-fill-active with the accent color failed to meet minimum contrast requirements for the accent colored counter text.  For consistency, I updated the community cards as well.

Reply:
I am completely fine with the color change that you have made here. What I am referring to is the importing of behaviors at the top of the page and the withBehaviors at the bottom of the page in those 2 sections you still have neutralFillActiveBehavior and it should be updated to neutralFillFocusBehavior if you are making this color change here.

Reply:
I gotcha.  All fixed up!